### DISCUSSION TITLE
How to make Table columns equal width?

### DISCUSSION THREAD

**Discussion by somguyth** [chakra-ui/chakra-ui#4446] [Q&A] [answered]
I'm looking at [this Table example](https://chakra-ui.com/docs/data-display/table#table-sizing) in the Chakra-UI docs. The middle column is much bigger in width than the other two. Is there a way to make all 3 columns of equal width/whitespace?

I was able to do it by supplying a manual, fixed width setting into `Tr`, but I prefer not to do it that way since it's not dynamic. I'd rather Chakra Table to automatically apply equal widths to each column?

See how wide the second column is:
![image](https://user-images.githubusercontent.com/37128022/126853120-28b89a5f-9e2c-4815-a106-014ef71438df.png)

**Comment by aaronkik** [comment, marked as answer]
Hi, you need apply css `table-layout: fixed` and a width value for your preffered size to get equal sized columns.

Such as:
```javascript
<Table __css={{'table-layout': 'fixed', width: 'full'}} variant="simple">
    // ...........
</Table>
```  

You can read more about table layout here: https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout
